woocommerce/woocommerce-ios
# Make sure test results are archived when test suites fail

### Description
The buildkite script stops running after test suite fails, but we still need the test result, well especially when any tests failed. This PR took [the approach](https://github.com/wordpress-mobile/WordPress-iOS/blob/4416fb9cc758fc6d9f650b16d535b7c34511b69b/.buildkite/commands/run-unit-tests.sh#L19-L22) used in WordPress-iOS repo

### Testing instructions
Make sure test results are archived when test failed. ~I'll update this description with screenshots of buildkite steps later.~

<img width="430" alt="Screen Shot 2022-09-05 at 1 07 39 PM" src="https://user-images.githubusercontent.com/1101828/188341827-cc02555b-dcb6-40eb-aef7-9a947ca64fa2.png">

### Screenshots
N/A


---
